config/locales/subscriptions.yml

```diff
@@ -21,7 +21,13 @@ en:
         daily: "You’ll get an email each day if a page is added or changed."
         weekly: "You’ll get one email per week if a page is added or changed."
     check_email:
-      title: Check your email inbox to confirm your subscription
-      description_html: "We’ve sent an email to: <strong>‘%{address}’</strong>"
-      instruction: Use the link in the email to confirm your subscription.
-      expiry: You have 7 days to confirm.
+      title: Check your email
+      description_html: "We’ve sent an email to <strong>‘%{address}’</strong>"
+      instruction: "Click the link in the email to confirm you want emails from GOV.UK about:"
+      expiry: The link will stop working after 7 days.
+      not_received_title: Not received an email?
+      not_received_detail_html: |
+        <p>Emails sometimes take a few minutes to arrive.</p>
+        <p>If you do not receive an email soon, check your spam or junk folder.</p>
+        <p>You may need to add gov.uk.email@notifications.service.gov.uk to your <a target="_blank" href="https://support.clickdimensions.com/hc/en-us/articles/115001160933-Add-a-Sender-to-the-Safe-Sender-List">safe sender list</a>.</p>
```

Comment:
Do we have precedent of using this link? It seems a pretty unusual destination to send people too, given that it's just some marketing company. I'd see if we could avoid the link and just have people Google it if we haven't got anything internal for this.

Reply:
I did surprise me a bit. @gavinwye any thoughts?

Reply:
Oh should the HTML here have govuk-frontend classes too: govuk-body and govuk-link ? or does the component have something to ensure these are styled correctly? It could be they look ok by default because of static but actually lack the styles they should have.

Reply:
Good point @kevindew 🕵️. It looks like it has [some of its own styles](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/details/_index.scss#L77). We also take [a similar approach in Content Publisher](https://github.com/alphagov/content-publisher/blob/master/app/views/components/_markdown_guidance.html.erb#L20).

Reply:
I've also removed the paragraph about safe senders, as discussed. @kevindew this should be ready for another review now.

Reply:
Nice digging thanks - not sure it's a great idea that we only style `<p>` elements. But alas, not todays problem.